arch/risc-v/src/mpfs/mpfs_can.c

```diff
@@ -678,7 +678,8 @@ static netpkt_t *mpfs_receive(struct netdev_lowerhalf_s *dev)
           if (pkt != NULL)
             {
               netpkt_copyin(dev, pkt, (uint8_t *)&cf,
-                           sizeof(struct can_frame), 0);
+                           sizeof(struct can_frame), 0,
+                           NETPKT_RX);
```

Comment:
Please align these lines. The sizeof and NETPKT_RX should start at same column as dev in the above line